observations/obs_converters/even_sphere/README.rst

```diff
@@ -2,75 +2,330 @@
+- the default date of the observations is 2017-12-25 00:00:00
+- a plot of the locations will be created. The number of gridlines is configurable but 
+  defaults to 288 in longitude and 192 in latitude.
+- **all** the defaults can be changed by specifying 'variable-value' pairs of options, 
+  as described below. Examples of some options are also available via the normal 
+  MATLAB *help* facility. (Documenting all of them in the *help* makes the help page too long.)
+
+Note that the number of observations will be the number of locations \* 
+the number of vertical levels \* the number of variables (i.e. 3) 
+*even_sphere.m* also takes observation error variances 
+and includes them in the observation sequences.
+
+Optional Argument Variable-Value pairs
+--------------------------------------
+
+The optional variable-value pairs can appear in any order.
+
++-------------------+----------------------------+--------------------------------------------------+
+| optional variable | example value              | Description                                      |
++===================+============================+==================================================+
+| 'nlevels'         | 14                         | number of pressure levels to use.                |
+|                   |                            | May be less than the length of the               |
+|                   |                            | 'levels' array, but cannot be more.              |
++-------------------+----------------------------+--------------------------------------------------+
+| levels            | [1000  500  300  200  100] | pressure levels desired.                         |
+|                   |                            | see `Levels`_ section for discussion.            |
++-------------------+----------------------------+--------------------------------------------------+
+| T_error_var       | [1.44 0.64 0.81 1.44 0.64] | level-specific                                   |
+|                   |                            | Temperature error variances.                     |
+|                   |                            | see `Levels`_ section for discussion.            |
++-------------------+----------------------------+--------------------------------------------------+
+| W_error_var       | [1.96 4.41 9.00 7.29 4.41] | level-specific error variances                   |
+|                   |                            | for both U, V wind components.                   |
+|                   |                            | see `Levels`_ section for discussion.            |
++-------------------+----------------------------+--------------------------------------------------+
+| 'YMD'             | '2017-12-25'               | Date required for *create_obs_sequence*.         |
+|                   |                            | This time is replaced if and when                |
```

Comment:
This might be clearer (and shorter):
"If *create_obs_sequence* is run, this time will be replaced."